docs/_static/auth/v1.yaml

```diff
@@ -146,12 +146,12 @@ paths:
         default:
           $ref: '../common/error_response.yaml'
 
-  /internal/auth/v1/bearertoken:
```

Comment:
There has been a request to be able to call the authorization server manually.
So it would be better to not replace the old, but just add a new API call.

Reply:
Yeah, wasn't sure about that. Thanks, I reverted removing the endpoint.